### PR TITLE
Importer docs: Update to emphasize running in CWD

### PR DESCRIPTION
**Pull request by MartinPetkov**

### CODE REVIEW

File: cmd/tfimport/README.md [hunk 1]
```diff
@@ -15,15 +15,23 @@ generated by the Terrarom Engine.
 
 ## Usage
 
+To run against the current working directory:
+
 ```shell
-go install ./cmd/tfimport
-tfimport --input_dir ~/path/to/terraform/configs/dir/
+go install github.com/GoogleCloudPlatform/healthcare-data-protection-suite/cmd/tfimport
+tfimport
```
Comment (umairidris) [inline cmd/tfimport/README.md:22]: To be clearer you can show two different methods

go install ./cmd/tfimport
cd ~/path/to/terraform/configs/dir
tfimport

go run ./cmd/tfimport --input_dir=~/path...

OR

Put a manfile- like section describing the flags. Then you don't need to provide examples for each flag.

File: cmd/tfimport/main.go [hunk 2]
```diff
@@ -36,8 +36,8 @@ import (
 )
 
 var (
-	inputDir      = flag.String("input_dir", ".", "Required. Path to Terraform config.")
-	terraformPath = flag.String("terraform_path", "terraform", "Optional. Name or path to the terraform binary to use.")
+	inputDir      = flag.String("input_dir", ".", "Path to Terraform config.")
```
Comment (umairidris) [inline cmd/tfimport/main.go:39]: Path to directory containing Terraform configs

File: cmd/tfimport/README.md [hunk 3]
```diff
@@ -15,32 +15,38 @@ generated by the Terrarom Engine.
...
 ## Usage
 
+### Installing and Running
+
+To install and run (recommended):
+
 ```shell
-go install ./cmd/tfimport
-tfimport --input_dir ~/path/to/terraform/configs/dir/
+go install github.com/GoogleCloudPlatform/healthcare-data-protection-suite/cmd/tfimport
```
Comment (umairidris) [inline cmd/tfimport/README.md:23]: To be clearer I still think we should show a dummy cd to show that we should be in the tf dir or put a comment.

Also, does `github.com/GoogleCloudPlatform/healthcare-data-protection-suite/cmd/tfimport` fetch the release at HEAD?

Comment (MartinPetkov) [reply cmd/tfimport/README.md:23]: It does (technically fetches the default branch, "master" in our case).

**Pull request merged**